### PR TITLE
Python add codebytes

### PR DESCRIPTION
**Pull request by karthikmurali60**
### Description

- Added Codebytes examples for three Python file methods: math.ceil, PyTorch and tqdm.
- Closes Issue https://github.com/Codecademy/docs/issues/2888

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by karthikmurali60** [comment]
@SSwiniarski @caupolicandiaz I have recreated the PR. I have omitted adding codebytes for pytorch since it was mentioned that it is not supported yet. Please review the PR when you get a chance !

**Comment by karthikmurali60** [comment]
@SSwiniarski @caupolicandiaz kind reminder to review this PR

**Comment by KTom101** [comment]
@SSwiniarski @caupolicandiaz I see you were already reviewing this in the previous PR so I will let you continue on.

File: content/python/concepts/math-module/terms/math-ceil/math-ceil.md [hunk 1]
```diff
@@ -43,3 +43,15 @@ import math
 
 print("The ceiling value of -16.7 is: ", math.ceil(-16.7))
 ```
+
+## Codebyte Example
+
```
Comment (SSwiniarski) [inline content/python/concepts/math-module/terms/math-ceil/math-ceil.md:48]: We usually have a sentence introducing the codebyte.

File: content/python/concepts/modules/terms/tqdm/tqdm.md [hunk 2]
```diff
@@ -45,3 +45,13 @@ print("done")
 This results in output like the following:
 
 ![tqdm Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/tqdm.gif)
+
+## Codebyte Example
+
```
Comment (SSwiniarski) [inline content/python/concepts/modules/terms/tqdm/tqdm.md:50]: We usually have a sentence introducing the codebyte.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@karthikmurali60 I just had a couple of comments.

**Comment by SSwiniarski** [comment]
@caupolicandiaz, I ran the tqdm codebyte in the codebyte editor, and while it did run, it didn't produce the output you'd expect in a real environment. Do we still want to include it?

**Comment by SSwiniarski** [comment]
@karthikmurali60 Would you mind removing the tqdm codebyte from this PR?

**Comment by karthikmurali60** [comment]
@SSwiniarski I have addressed your review comments and also removed the tqdm codebyte from this PR. Please review when you get a chance. Thanks

**Review by SSwiniarski — APPROVED** [review]
@karthikmurali60 Looks good.

**Pull request merged**